soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -575,7 +575,7 @@ private class Merger[T](andFunction: MonomorphicFunction[T]) {
   // for a query that contains a search.
   private def tryMerge(a: Analysis, b: Analysis): Option[Analysis] = (a, b) match {
     case (a, _) if (hasWindowFunction(a)) => None
-    case (SoQLAnalysis(aIsGroup, false, aSelect, Nil, aWhere, aGroup, aHaving, aOrder, aLim, aOff, aSearch),
+    case (SoQLAnalysis(aIsGroup, false, aSelect, Nil, aWhere, aGroup, aHaving, aOrder, aLim, aOff, None),
```

Comment:
Are we adding the search to the query elsewhere? It looks like all of these just remove it, I expect I'm missing something though.

Reply:
These search cases just won't match any more and go to the catch all case which means not mergeable.  Then the chained soql retains it's original form with search.

Reply:
These search cases just won't match any more and go to the catch all case which means not mergeable.  Then the chained soql retains it's original form with search.